fulp_modules/jobs/deputy/code/deputy_grabbing.dm

```diff
@@ -0,0 +1,41 @@
+/datum/martial_art/deputygrab
+	name = "Deputy Grab"
+	id = MARTIALART_DEPUTYGRAB
+	var/old_grab_state = null
+	var/in_department = TRUE
+	var/list/valid_area = null
+
+/datum/martial_art/deputygrab/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D) // From CQC.dm
+	if(!can_use(A))
+		return FALSE
+	if(in_department)
```

Comment:
what is the purpose of this? you never change it, itll always stay at true and you check for the area a line down

Reply:
I wasnt entirely sure how the file worked, I put can_use as a way to get their valid_area, then in_department is what checks to see if they are in said area